semantic-tsx/src/Language/TSX/Tags.hs

```diff
@@ -295,7 +308,7 @@ instance ToTags Tsx.Undefined
 instance ToTags Tsx.UnionType
 instance ToTags Tsx.UpdateExpression
 instance ToTags Tsx.VariableDeclaration
-instance ToTags Tsx.VariableDeclarator
+-- instance ToTags Tsx.VariableDeclarator
```

Comment:
Looks like we can 🔥 this now

Reply:
The pattern I've been using is to comment them out since generating the full list to begin with was automated and pasted in. @robrix do you have an opinion here?